src/org/nfrac/comportex/cells.cljc

```diff
@@ -1078,19 +1099,16 @@
   (predictive-cells [_]
     (when (== (:timestep state)
               (:timestep distal-state))
-      (set/union (:pred-cells distal-state)
-                 (:pred-cells apical-state))))
+      (:pred-cells distal-state)))
```

Comment:
Maybe yeah. Not sure what all the uses cases are for the predictive-cells set. (It's not used in the algorithm).